src/openforms/js/components/admin/form_design/registrations/json_dump/fields/Variables.js

```diff
@@ -0,0 +1,43 @@
+import {useField} from 'formik';
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Field from 'components/admin/forms/Field';
+import FormRow from 'components/admin/forms/FormRow';
+import VariableSelection from 'components/admin/forms/VariableSelection';
+
+const Variables = () => {
+  const [fieldProps] = useField('variables');
+
+  return (
+    <FormRow>
+      <Field
+        name="variables"
+        label={
+          <FormattedMessage
+            description="JSON registration options 'variables' label"
+            defaultMessage="Variables"
+          />
+        }
+        helpText={
+          <FormattedMessage
+            description="JSON registration options 'variables' helpText"
+            defaultMessage="Which variables to include in the data to be sent"
+          />
+        }
+        required
+        noManageChildProps
+      >
+        <VariableSelection
+          {...fieldProps}
+          isMulti
+          required
+          closeMenuOnSelect={false}
+          includeStaticVariables={true}
```

Comment:
```suggestion
          includeStaticVariables
```
this is equivalent in JSX - no need to change this now, but for future code this is the idiomatic way to specify true boolean prop values